### PR TITLE
Replace smilies with better versions, refresh comments list after new comment is saved

### PR DESCRIPTION
**Pull request by iantrudell**
Feel free to modify this (need to add update SQL I think for the smilies). The PNG versions are transparent and improve on the GIF versions.

For the refresh comments list part, the objective is to reload/refresh the comments list after a new comment is submitted/saved. Currently, it doesn't always update the comment lists and users frequently double-post as a consequence.

### CODE REVIEW

**Comment by exstreme** [comment]
1) May be better disable button?
2) How about delay with another message (or yoy set this value to 0 in settings?)
3) Can you describe, how to repeat this case?

**Comment by iantrudell** [comment]
> 1. May be better disable button?
> 2. How about delay with another message (or yoy set this value to 0 in settings?)
> 3. Can you describe, how to repeat this case?

1. Not sure what you mean by disable button
2. Not sure what you mean
3. Load a page with many comments, then submit a new comment. Currently, it won't reliably/automatically refresh the comments list to show the newly posted comment. With this change, it will always refresh the comments list after a user adds a new comment.

**Comment by exstreme** [comment]
@iantrudell may be can you share link, where we can reproduce and view the bug?
And your branch has conflicts

**Comment by Globulopolis** [comment]
> 3\. Load a page with many comments, then submit a new comment. Currently, it won't reliably/automatically refresh the comments list to show the newly posted comment. With this change, it will always refresh the comments list after a user adds a new comment.

I have seen this bug when comments displayed as list view(not a tree). List won't update(and no errors).

File: component/media/js/jcomments-v4.0.js [hunk 1]
```diff
@@ -1473,63 +1483,56 @@ JComments.prototype = {
...
-					$this.attr({
-						href: _response.data.href,
-						title: _response.data.title
-					});
-					$this.html('<span aria-hidden="true" class="icon-mail icon-fw"></span> ' + _response.data.title);
 
-					Joomla.renderMessages({'message': [_response.message]}, '.comments-list-footer');
-				} else {
-					Joomla.renderMessages({'warning': [_response.message]}, '.comments-list-footer');
+	document.querySelectorAll('.cmd-subscribe').forEach(function (subscribeButton) {
```
Comment (Globulopolis) [inline component/media/js/jcomments-v4.0.js:1514]: Only one link to subscribe in document. Use querySelector() instead.

Comment (iantrudell) [reply component/media/js/jcomments-v4.0.js]: Fixed!

**Comment by exstreme** [comment]
Is it 18x18 to small resolution in 2024?
And gif can have animation and can be transparent too!
SQL usn't updatable, only for new installs

**Pull request merged**